\hypertarget{classpip_1_1__vendor_1_1pep517_1_1in__process_1_1__in__process_1_1__DummyException}{}\doxysection{pip.\+\_\+vendor.\+pep517.\+in\+\_\+process.\+\_\+in\+\_\+process.\+\_\+\+Dummy\+Exception Class Reference}
\label{classpip_1_1__vendor_1_1pep517_1_1in__process_1_1__in__process_1_1__DummyException}\index{pip.\_vendor.pep517.in\_process.\_in\_process.\_DummyException@{pip.\_vendor.pep517.in\_process.\_in\_process.\_DummyException}}


\doxysubsection{Detailed Description}
\begin{DoxyVerb}Nothing should ever raise this exception\end{DoxyVerb}
 Inheritance diagram for pip.\+\_\+vendor.\+pep517.\+in\+\_\+process.\+\_\+in\+\_\+process.\+\_\+\+Dummy\+Exception\+:\begin{figure}[H]
\begin{center}
\leavevmode
\includegraphics[height=2.000000cm]{classpip_1_1__vendor_1_1pep517_1_1in__process_1_1__in__process_1_1__DummyException}
\end{center}
\end{figure}


The documentation for this class was generated from the following file\+:\begin{DoxyCompactItemize}
\item 
venv/\+Lib/site-\/packages/pip/\+\_\+vendor/pep517/in\+\_\+process/\+\_\+in\+\_\+process.\+py\end{DoxyCompactItemize}
